test/fixtures/elastic-responses/example-get-response-object2.json

```diff
@@ -0,0 +1,443 @@
+        "purpose": "conservation"
+      },
+      {
+        "id": "625747",
+        "locations": [
+          {
+            "primary": true,
+            "type": "location",
+            "value": "No location available"
+          }
+        ],
+        "objid": "205752"
+      },
+      {
+        "id": "782580",
+        "locations": [
+          {
+            "primary": true,
+            "type": "location",
+            "value": "National Railway Museum"
+          },
+          {
+            "type": "last location",
+            "value": "No location available"
+          }
+        ],
+        "objid": "205752",
+        "purpose": "conservation"
+      },
+      {
+        "id": "860263",
+        "locations": [
+          {
+            "primary": true,
+            "type": "location",
+            "value": "National Railway Museum, Great Hall"
+          },
+          {
+            "type": "last location",
+            "value": "National Railway Museum"
+          }
+        ],
+        "objid": "205752",
+        "purpose": "on display"
+      }
+    ],
+    "name": [
+      {
+        "primary": true,
+        "type": "object name",
+        "value": "steam locomotive"
+      }
+    ],
+    "numbers": {
+      "NUMBER1": "401"
+    },
+    "options": {
+      "option3": "Coulls, Anthony"
+    },
+    "summary_title": "LMS(R) steam locomotive with tender 'Duchess of Hamilton' 4-6-2 Coronation class, No 6229, 1938 (steam locomotive)",
+    "terms": [
+      {
+        "@link": {
+          "role": [
+            {
+              "value": "NRM Category"
+            }
+          ],
+          "type": "reference"
+        },
+        "admin": {
+          "id": "term-266627",
+          "uid": "smgc-term-266627"
+        },
+        "summary_title": "Locomotive and Rolling Stock"
+      },
+      {
+        "@link": {
+          "role": [
+            {
+              "value": "NRM Category filter"
+            }
+          ],
+          "type": "reference"
+        },
+        "admin": {
+          "id": "term-266835",
+          "uid": "smgc-term-266835"
+        },
+        "summary_title": "steam"
+      },
+      {
+        "@link": {
+          "role": [
+            {
+              "value": "Object Name"
+            }
+          ],
+          "type": "reference"
+        },
+        "admin": {
+          "id": "term-502655",
+          "uid": "smgc-term-502655"
+        },
+        "summary_title": "steam locomotive"
+      }
+    ],
+    "title": [
+      {
+        "primary": true,
+        "type": "display title",
+        "value": "LMS(R) steam locomotive with tender 'Duchess of Hamilton' 4-6-2 Coronation class, No 6229, 1938"
+      },
+      {
+        "type": "display title",
+        "value": "LMS(R) steam locomotive with tender 'Duchess of Hamilton' 4-6-2 Coronation class, No 46229, 1938"
+      }
+    ],
+    "type": {
+      "base": "object",
+      "record_type": "WHOLE_PART"
+    }
+  }
+}
```

Comment:
@SimonLab _many_ people I've worked with prefer a "_known state_" for their test/fixture data.
(_I personally prefer to use **real data** because it tells us quickly if something is "broken" ... see: https://github.com/numo-labs/lambda-ne-classic-package-provider/blob/3d2e649c87237d269cf62c8e2c12eb0e8a847d37/package.json#L7 where we create our fixtures from the **acutal** API/data before running our tests_)